tests/fixtures/query_sanitazion/swagger.yaml

```diff
@@ -1,22 +0,0 @@
-swagger: "2.0"
```

Comment:
I don't think this should be removed.

Reply:
I believe that this file is only used by the `query_sanitazion` fixture which, if I am looking correctly, is also unused. Therefore, I have removed them.

In order to test that sanitization of request body is indeed disabled in both OpenAPI v2 and OpenAPI v3, the "simple" test application has been extended with a `forward` endpoint that is used in new test `test_no_sanitization_in_request_body`.

Reply:
I think there's a miscommunication - sorry if I was unclear.
* form-data sanitization still needs to happen in swagger2 (you haven't changed this)
* this fixture tests form-data sanitization for swagger2
* by removing this fixture, you've removed form-data sanitization testing for swagger2
* the test you added is related to a body with content-type: application/json

so, I think we still want coverage of form-data sanitization for swagger2.

hopefully this makes sense - let me know if I missed something.

Reply:
It's unfortunate that the coveralls plugin was removed from this repo - it would help to directly illustrate what I'm talking about

Reply:
> form-data sanitization still needs to happen in swagger2 (you haven't changed this)

I agree.

> this fixture tests form-data sanitization for swagger2

Yes, but that fixture is unused, if I am looking correctly.

I can try adding a new test for the sanitization of form data tomorrow, but that would be out of the original scope of this MR.

Reply:
Ah - I see the disconnect - apparently we've _already_ lost test coverage of the sanitization somewhere along the way, and you are just removing an unused fixture.
My mistake - I thought that you had removed a test as well.
I can see how from your perspective it doesn't make sense to add that test to this merge request.

Reply:
I have looked at the current code and there already seems to be a test [`test_param_sanitization`](https://github.com/zalando/connexion/blob/master/tests/api/test_parameters.py#L368) which checks the sanitization of form data parameters in both OpenAPI v2 and OpenAPI v3.

It could be improved and extended, but I would not do it within this MR because it is in my opinion out of scope.

Reply:
ok, that makes sense - thanks!